.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: CI workflow
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 11.x, 12.x, 13.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm install
```

```suggestion
      run: npm install --ignore-scripts
```

I'm spreading this install script 👍